manifests/g/Gitbutler/GitButler/0.11.0/Gitbutler.GitButler.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Gitbutler.GitButler
+PackageVersion: 0.11.0
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{7B797A7C-C01B-4219-9190-5905FA70DB80}'
+Installers:
+- InstallerUrl: https://app.gitbutler.com/downloads/release/windows/x86_64/msi

```

```suggestion
- InstallerUrl: https://releases.gitbutler.com/releases/release/0.11.0-810/windows/x86_64/GitButler_0.11.0_x64_en-US.msi
```
Versioned URLs are preferred